src/main/java/Calculator.java

```diff
@@ -0,0 +1,65 @@
+import java.util.Scanner;
+
+public class Calculator {
+
+    public int result(String text){
+        if(isBlank(text)){
+            throw new IllegalArgumentException("빈 값을 입력받을 수 없습니다.");
+        }
+        int result = strToIntCalculator(splitString(text));
+        return result;
+    }
+
+    public boolean isBlank(String text){
+        if (text == null || text.isEmpty())
+            return true;
+        return false;
+    }
```

Comment:
```suggestion
    public boolean isBlank(String text){
        return text == null || text.isEmpty();
    }
```
이렇게 변경해 볼 수 있을 것 같아 :) 그런데 isBlank를 메서드로 분리한 이유가 뭐야??

Reply:
최대한 기능별로 세분화해야 한다는 강박에 따로 뺀 거였어! 근데 앞에서 언니가 알려준대로 접근제어자를 private로 변경해보니까 테스트코드 작성하는데도 오류가 있더라구. 찾아보니까 프라이빗 메소드를 테스트를 할땐 Reflaction(추천하지 않는 방법)을 사용하거나 퍼블릭 메소드를 이용하면 된다고 하는데, 계산기 프로그램에서 중요한 기능도 아니고, 기능이라기 보다는 조건에 가깝다고 생각돼서 isBlank 메소드랑 테스트코드는 삭제했어!

그리구 언니가 수정해준 코드 보고 더 간단하게 할수있구나 유레카를 외쳤습니다 다른 메소드 만들때도 더 간단하고 깔끔하게 짤 수 있게 계속 생각하면서 해볼게ㅎㅎ